vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
Do you know why this change was included? When I run trash this line gets added back. It's not a big deal either way since it'll just get changed back whenever the vendor folder is updated again.